symphony/content/content.blueprintssections.php

```diff
@@ -967,12 +967,12 @@ protected function validateTimestamp($section_id, $checkMissing = false)
     {
         if (!isset($_POST['action']['ignore-timestamp'])) {
             if ($checkMissing && !isset($_POST['action']['timestamp'])) {
-                $this->_errors['timestamp'] = __('The timestamp is missing.');
+                $this->_errors['timestamp'] = __('Saving aborted, timestamp is missing.');
                 return false;
             } else if (isset($_POST['action']['timestamp'])) {
                 $tv = new TimestampValidator('sections');
                 if (!$tv->check($section_id, $_POST['action']['timestamp'])) {
-                    $this->_errors['timestamp'] = __('The action requested was made on a earlier version of the record.');
+                    $this->_errors['timestamp'] = __('Saving aborted, data integrity check failed.');
```

Comment:
"data integrity check failed" isn't very user-friendly, is it? Can we make this more clear?